### ISSUE TITLE
Add option to disable Squiz.Commenting.FunctionComment.ScalarTypeHintMissing

### ISSUE THREAD

**Issue by Vrtak-CZ** [squizlabs/PHP_CodeSniffer#870]
If we have code with is written primary for PHP 5.6 but PHP_CodeSniffer is run on PHP 7.0 it reports "false positive" _Squiz.Commenting.FunctionComment.ScalarTypeHintMissing_ (because code is written for PHP 5.6).

We have PHP 7.0 on our CI server because we are working on upgrade to PHP 7.0 on our production server with is running on PHP 5.6. So we need PHP 5.6 compatible code.

**Comment by gsherwood** [comment]
Hopefully you're familiar with ruleset.xml files. If so, you'll need to add this to mute that error:
```xml
<rule ref="Squiz.Commenting.FunctionComment.ScalarTypeHintMissing">
  <severity>0</severity>
</rule>
```
Or if you already have an exclude list, use this (it does the same thing as the above code):
```xml
<exclude name="Squiz.Commenting.FunctionComment.ScalarTypeHintMissing"/>
```
If you don't know what PHPCS rulesets are, have a read about them here: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml

You can also create a file called `phpcs.xml` (or `phpcs.xml.dist`) in your project's root directory and use this as a project-wide ruleset. Then just running `phpcs` on some code will find and use this custom ruleset for you.

**Comment by Vrtak-CZ** [comment]
Thank you!

```xml
<exclude name="Squiz.Commenting.FunctionComment.ScalarTypeHintMissing"/>
```
works :-)

**Comment by gsherwood** [comment]
No problem. Glad it worked for you.

**Comment by sergey-rud** [comment]
Do you have any suggestion for a case when you inherit interface from 3rd party package and you cannot add type hint because it will break compatibility with the interface?

Having something similar to `@phpcsSuppress SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingParameterTypeHint` would be excellent.

**Comment by jrfnl** [comment]
@sergey-rud Try `// phpcs:ignore Squiz.Commenting.FunctionComment.ScalarTypeHintMissing` on the line above it or adding `// @phpcs:disable Squiz.Commenting.FunctionComment.ScalarTypeHintMissing` to the function docblock and `// phpcs:enable` just below the function signature line.

Ref: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file
